### DISCUSSION TITLE
Dicebear exporter not exporting properties/components

### DISCUSSION THREAD

**Discussion by ahelminiak-rdg** [dicebear/dicebear#436] [Q&A] [answered]
I have been trying various ways to get the Figma plugin to work properly, but all I end up with is an index file and empty properties, so nothing ever changes beyond the default. I'm not sure if this is a plugin bug or something I'm doing wrong in Figma, but I would appreciate some insights for things to try/troubleshoot, or maybe often overlooked or incorrect settings? I have my components grouped properly I believe, and placed one of each on a square frame (can be changed to different versions using the Figma dropdown). But after export, there is only an index.ts file, no file per component or anything, nothing changes after a build, etc.

Here's what the schema looks like (empty properties):
```
import type { StyleSchema } from '@dicebear/core';

export const schema: StyleSchema = {
  "$schema": "http://json-schema.org/draft-07/schema#",
  "properties": {}
};
```
Help please!

**Comment by FlorianKoerner** [comment]
Hi @ahelminiak-rdg 

In order to help you effectively, I need access to the file. Please send me a link to contact@dicebear.com

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @ahelminiak-rdg 

Thank you for sending me your file. I have taken a look at it. The components must not be in a frame, but must be directly in the root. Like so:

![Bildschirmfoto 2024-10-04 um 16 39 18](https://github.com/user-attachments/assets/713b7cbc-762b-4942-8db9-0e5a6b9de0db)

**Comment by ahelminiak-rdg** [comment]
Oh my, so simple! Thank you! I was looking at someone else’s file for reference and they had theirs in a frame, so I was thrown off. It works now!

Ashley Helminiak
Rapid Development Group
***@***.*** ***@***.***> | 616.540.6192
